books/free-programming-books-subjects.md

```diff
@@ -876,6 +876,7 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Web Services
 
 * [RESTful Web Services](http://restfulwebapis.org/RESTful_Web_Services.pdf) (PDF)
+* [TypeScript Notes for Professionals](https://books.goalkicker.com/TypeScriptBook2/)
```

Comment:
Could yo move this to `books-langs`? You select a wrong category

Reply:
yes